database-sharding.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: 'What is horizontal sharding?'
+description: 'A basic overview of horizontal sharding. What is it and what are the challenges?'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/database-sharding-min.png
+  alt: Quick read on horizontal sharding
+author: J Edwards
+date: 11-23-2023
+published: true
+slug: xatabyte-horizontal-sharding
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/database-sharding-twitter-og-min.png
+---
+
+> Want to grasp a complex concept in the time it takes to brew your morning coffee? In our **XataByte: 3-minute byte-sized read** series we offer concise explanations to help you unravel intricate database concepts without the time commitment. Today we are looking at **horizontal sharding**.
```

Comment:
```suggestion
> Want to grasp a complex concept in the time it takes to brew your morning coffee? In our **XataByte: 5-minute byte-sized read** series we offer concise explanations to help you unravel intricate database concepts without the time commitment. Today we are looking at **horizontal sharding**.
```

The article has robust content and is definitely not a 3-minute read! 5 minute is also pushing it but I think it makes makes the point without overpromising on how short it is :)